packages/charting/src/components/LineChart/LineChart.types.ts

```diff
@@ -0,0 +1,139 @@
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
+
+export interface ILineChart {}
+
+export interface IDataPoint {
+  /**
+   * Independent value of the data point, rendered along the x-axis.
+   * If x is a number, then each y-coordinate is plotted at its x-coordinate.
+   * If x is a string, then the data is evenly spaced along the x-axis.
+   */
+  x: number | string;
+
+  /**
+   * Dependent value of the data point, rendered along the y-axis.
+   */
+  y: number;
+}
+
+export interface ILineChartProps {
+  /**
+   * Data to render in the chart.
+   */
+  data?: IDataPoint[];
+
+  /**
+   * Width of the chart.
+   */
+  width?: number;
+
+  /**
+   * Height of the chart.
+   */
+  height?: number;
+
+  /**
+   * Number of ticks on the y-axis.
+   */
+  yAxisTickCount?: number;
+
+  /**
+   * Colors from which to select the color of each line.
```

Comment:
Nit: this comment isn't entirely accurate since the `color` here is one string that applies to every line rather than an array of colors that generates a gradient (as was the case for the `VerticalBarChart`.